README.md

```diff
@@ -21,6 +21,22 @@ and review the napari docs for plugin developers:
 https://napari.org/plugins/stable/index.html
 -->
 
+This plugin uses [scikit-image's blob detection alrogithms](https://scikit-image.org/docs/stable/auto_examples/features_detection/plot_blob.html) to detect bright blobs on dark backgrounds.
+
+Parameters:
```

Comment:
This is mostly a copy of the docstrings for the `detect_blobs` right?

I think it would be better to have a short description of what this plugin does instead.

```
It accepts a single image layer with two, three, or more dimensions, and outputs a points layer of equal dimensionality that contains the detected features.
```

It would also be great to have a short animation of running this in napari, even with a synthetic dataset, plus a short code example.